nbbuild/cluster.properties

```diff
@@ -445,8 +445,10 @@ nb.cluster.ide=\
         o.eclipse.mylyn.wikitext.core,\
         o.eclipse.mylyn.wikitext.markdown.core,\
         o.eclipse.mylyn.wikitext.textile.core,\
+        o.jboss.spec.javax.servlet,\
```

Comment:
Why do we need this?

Reply:
JGit 4.11.0 has an OSGi dependency on the Servlet API 3.1, and the only package I found on Maven Central that provided this requirement is the one from JBoss!

Click here for details: 

https://github.com/eclipse/jgit/blob/61e4f1665221626cfd31a5826bff527ef7ce5719/pom.xml#L209
https://github.com/eclipse/jgit/blob/stable-4.11/org.eclipse.jgit/META-INF/MANIFEST.MF

Reply:
... but why does JGit have a dependency on the Servlet API? I found this https://github.com/eclipse/jgit/commit/6087031469487e23b5e7055d0d4e65e2ce96fcbd#diff-66e59ab180153a142ed3da762d562591 which tries to remove the dependency. Is it used somewhere? Do we even need the code that has that dependency?

Reply:
Indeed, that's a very good question! I can ask the project maintainer if you want?

Reply:
I see that it was removed in 4.7.x (the link you provided) but has been reintroduced in 4.11.x:
https://github.com/eclipse/jgit/commit/94bcde663c75735049aae0acbd9f6d8519ed1f05#diff-66e59ab180153a142ed3da762d562591

Reply:
I contacted the author and he confirmed that it was a mistake! 

He fixed it already: https://git.eclipse.org/r/#/c/120220/